packages/components/TechSection.tsx

```diff
@@ -46,14 +46,15 @@ function TechSection() {
 
   return (
     <Section>
-      <Heading my="4rem" size="md" fontWeight={600} textAlign="left">
-        {t("tech-title")}
-      </Heading>
-      <HStack spacing="1rem" flexWrap="wrap" justifyContent="center" alignItems="center" lineHeight="200%">
-        {iconList.map((Icon) => (
-          <Icon p="0.3rem" key={Icon.name} />
-        ))}
-      </HStack>
+      <h2 className="mx-4 my-16 font-semibold text-left text-xl leading-tight">{t("tech-title")}</h2>
```

Comment:
It seems the styles are not looking good, as we have a huge dead-space:

https://github.com/user-attachments/assets/62f71099-df90-40a1-a2f0-bbaa76444769

Let's try to improve this by:
- add styles to `Section` to add a `gap-16` as a style
- remove `my-16` from text
- I also suggest removing the `text-left`, as it's kind of weird the content centralized, and the text on the left side

Reply:
So, I also noticed all this space; but since the task is focused on migrating the component while keeping the same structure and style, I didn't try to change it.

As you can see below, it has the same space:

![image](https://github.com/user-attachments/assets/bfa9dae1-0a8f-4e7e-8a90-79be4fa42c1f)

Reply:
@marco-souza For me, it would be beyond the scope of the task to change the structure/style of the component. What do you think?

Reply:
@Diegiwg I agree that this minor change is not part of the migration scope.

When I encounter these kinds of quick wins and I'm working in the same context (same file or component), I typically apply the Scout Law, which emphasizes the importance of leaving things better than you found them.

In other words: It would be nice if we do this here, or else you will need to:
1. creating a new ticket
2. creating a new PR 
3. asking for review

> !NOTE:
> I would not do the same for big refactoring, only minor changes

Reply:
Reduced "dead" space.

![image](https://github.com/user-attachments/assets/4dbc9a49-e1f5-4a14-ad18-3f3001eab939)